raiden/constants.py

```diff
@@ -10,6 +10,8 @@
 UINT256_MAX = 2 ** 256 - 1
 UINT64_MAX = 2 ** 64 - 1
 
+FIRST_BLOCK_NUMBER = 0
```

Comment:
Nitpick: Wouldn't it make this more obvious if we call it `GENESIS_BLOCK_NUMBER`?